src/ai/backend/manager/models/scaling_group.py

```diff
@@ -89,6 +91,9 @@ def as_trafaret(cls) -> t.Trafaret:
                 t.Key("pending_timeout", default=0): tx.TimeDuration(allow_negative=False),
                 # Each scheduler impl refers an additional "config" key.
                 t.Key("config", default={}): t.Mapping(t.String, t.Any),
+                t.Key("agent_selection_strategy", default=AgentSelectionStrategy.DISPERSED): t.Enum(
+                    *[e.value for e in AgentSelectionStrategy]
+                ),
```

Comment:
```suggestion
                t.Key("agent_selection_strategy", default=AgentSelectionStrategy.DISPERSED): tx.Enum(
                    AgentSelectionStrategy
                ),
```

we have [customized Enum trafaret type](https://github.com/lablup/backend.ai/blob/main/src/ai/backend/common/validators.py#L212). Let's use this!